cfgov/prepaid_agreements/jinja2/prepaid_agreements/detail.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/prepaid-agreements/css/main.css') }}">
+{%- endblock css %}
+
+{% block title -%}
+    {{ product.name }} | Consumer Financial Protection Bureau
+{%- endblock title %}
+
+{% block hero %}
+<div class="wrapper content_wrapper">
+
+    {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
+    {% set breadcrumb_items = [{
+        'href': '/prepaid-agreements/',
+        'title': 'Prepaid Product Agreements Database'
+    }]%}
+    {{ breadcrumbs.render(breadcrumb_items) }}
+</div>
+{% endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom prepaid-agreements prepaid-agreements-detail
+{%- endblock %}
+
+{% block content_main %}
+
+{% set default_text = 'No information provided' %}
+<h1>{{ product.name }}</h1>
+<div class="content-l">
+        <div class="content-l_col content-l_col-1-2">
+            <dl>
+                <dt><h2>Issuer name</h2></dt>
+                <dd>{{ product.issuer_name }}</dd>
+                <dt><h2>Prepaid product type</h2></dt>
+                <dd>{{ product.prepaid_type if product.prepaid_type else default_text }}</dd>
+            </dl>
+        </div>
+        <div class="content-l_col content-l_col-1-2">
+            <dl>
+                <dt><h2>Program manager</h2></dt>
+                <dd>{{ product.program_manager if product.program_manager else default_text }}</dd>
+                <dt><h2>Current status</h2></dt>
+                <dd>{{ product.status if product.status else default_text }}</dd>
+            </dl>
+        </div>
+    </div>
+        <dl>
+            <dt><h2>Other relevant parties</h2></dt>
+            <dd>
+            {% if product.other_relevant_parties %}
+                {% if product.other_relevant_parties | length > 500 %}
+                    <div data-qa-hook="expandable"
+                         class="o-expandable">
+                        <div class="o-expandable_content">
+                {% endif%}
+                {{ product.other_relevant_parties | replace('\n', '; ') }}
+                {% if product.other_relevant_parties | length > 500 %}
```

Comment:
How to test showing this condition?

Reply:
Here's a product that meets the condition: [Global Cash Card Mastercard](http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/49606/)